content/Dev/remote_settings_history.rst

```diff
@@ -0,0 +1,179 @@
+
+Alexis, Rémy, and Tarek were building Web APIs in the Cloud Services team, and there were needs to offer remote storage features to `Firefox OS <https://en.wikipedia.org/wiki/Firefox_OS>`_ app developers.
+
+I was hired in their team and we were about to roll out our first app: a «reading list» service (à-la Pocket).
+
+
+Neolithic (2015-2016)
+=====================
+
+The team had accumulated a lot of experience with Web APIs, and one thing was certain: we shouldn't start each project from scratch. Even though we had a secret plan (a reusable service), we couldn't justify starting with that.
+
+We gathered a lot of good practices and a handful tooling in one library: `Cliquet <https://cliquet.readthedocs.io/en/latest/>`_. We hated the word «framework», but at least we wanted a standard way of building services (cf. `talk at PyBCN <https://mozilla-services.github.io/cliquet/talks/2015.07.pybcn/>`_).
+
+.. image:: images/remote-settings-history-cliquet.png
+
+*Cliquet* had the notion of pluggable storage backends. And, because we had this vision of reusable services, we built a proof-of-concept where a service built with *Cliquet* would be storing its data in another service built with *Cliquet* 🤯 [#]_.
+
+Eventually, this storage service became `Kinto <https://github.com/Kinto/>`_, and *Cliquet* became ``kinto.core``.
+
+`n1k0 <https://nicolas.perriault.net/>`_ and `Magopian <https://mathieu.agopian.info/>`_ joined the team, and built a nice offline-first SDK for Kinto. Plus a generic UI to CRUD records of any accessible Kinto server (kind of remote phpmyadmin). In order to generate forms from collections JSON schema, they released `react-jsonschema-form <https://github.com/rjsf-team/react-jsonschema-form/>`_ (which now has like 9k stars on Github!).
+
+This was an exciting time! Kinto was on Hackernews [#]_ [#]_, and the number of Github stars was going up!
+
+
+At this time, there were many other challengers in the field of offline-first apps and backends as a service, and we were pretty close to them. `Michiel de Jong <https://michielbdejong.com/>`_, the spec author of `Remote Storage <https://remotestorage.io/>`_, joined Mozilla and was briefly in our team. `Dale Harvey <https://arandomurl.com/>`_ was already at Mozilla and was doing `PouchDB <https://pouchdb.com/>`_. `Luca Marchesini <https://github.com/xbill82/>`_, a Fullstack Fest friend, was also working on `Kuzzle <https://kuzzle.io/>`_.
+
+But internally in the company — above Tarek (our manager) — we were not able to defend our vision very well. Mozilla didn't seem ready to explore the dangerous grounds of hosting user data, at least not beyond *Firefox Sync*. And meanwhile, *Kinto* was still being mentioned as the Mozilla response to Facebook Parse and Google Firebase (completely irrelevant of course).
```

Comment:
You may want to add a sentence or two explaining why this comparison was irrelevant.